verilog/analysis/checkers/always_ff_non_blocking_rule.h

```diff
@@ -48,7 +50,26 @@ class AlwaysFFNonBlockingRule : public verible::SyntaxTreeLintRule {
   // Diagnostic message.
   static const char kMessage[];
 
+  // Collected violations.
   std::set<verible::LintViolation> violations_;
+
+  //- Configuration ---------------------
+  bool catch_modifying_assignments_ = false;
+  bool waive_for_locals_ = false;
+
+  //- Processing State ------------------
+  // Inside an always_ff block
+  int inside_ = 0;
+
+  // Stack of inner begin-end scopes
+  using Scope =
+      std::pair<int, int>;  // depth in syntax tree, number of inherited locals
```

Comment:
Instead of a pair, it might be as easy to create a struct here with the advantage. Something like:
```
struct Scope { int syntax_tree_depth; int inherited_locals_count; };
```

Self explantory and at the place where it is used, there is no wondering what first or second means.